### PR TITLE
Bypass template processing grc

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/18218

### CODE REVIEW

File: governance/custom_template.adoc [hunk 1]
```diff
@@ -453,3 +454,12 @@ See the following table for a comparison of hub cluster and managed cluster temp
 |===
 
 
+[#disable-template-processing]
+=== Disable template processing
+
+You might create a policy that contains a template that is not intended to be processed by {product-title-short}. By default, {product-title-short} processses all templates. 
+
+To disable template processing for your hub cluster, you must change `{{ template content }}` to `{{ '{{ template content }}' }}`.
```
Comment (gparvin) [inline governance/custom_template.adoc]: I have a small concern here that this wording sounds like the template processing is being disabled for the entire hub cluster.  The intent is for the user to modify the template so it does not get processed by ACM.  You need to modify each template in the policy that you don't want ACM to process in this way (or you do the alternative workaround that is mentioned after this)

**Review by gparvin — COMMENTED** [review]
See comment on line 462 about a phrase that could be confusing.

File: governance/custom_template.adoc (hunk 1 above)
Comment (dockerymick) [reply governance/custom_template.adoc]: @gparvin Thanks Gus for your feedback! I made some updates. When you have a chance, let me know your thoughts

File: governance/custom_template.adoc [hunk 2]
```diff
@@ -453,3 +454,12 @@ See the following table for a comparison of hub cluster and managed cluster temp
 |===
 
 
+[#modify-template-processing]
+=== Modify template processing
+
+You might create a policy that contains a template that is not intended to be processed by {product-title-short}. By default, {product-title-short} processses all templates. 
+
+To modify template processing for your hub cluster, you must change `{{ template content }}` to `{{ '{{ template content }}' }}`.
```
Comment (gparvin) [inline governance/custom_template.adoc]: Would it be ok to say:
To bypass {product-title-short}'s template processing, you must ...

**Review by gparvin — COMMENTED** [review]
I'm not sure if "bypass" is an acceptable word to use, but I think something more like the suggestion I added would help make it a little more clear what we're doing.

File: governance/custom_template.adoc (hunk 2 above)
Comment (dockerymick) [reply governance/custom_template.adoc]: Yea, I think that its fine to say! I checked the IBM Style Guide and there were no mentions of it. Making the update now

**Review by gparvin — APPROVED** [review]

**Pull request merged**